backend/danswer/direct_qa/qa_prompts.py

```diff
@@ -100,46 +194,6 @@ def freeform_processor(question: str, documents: list[str]) -> str:
     return prompt
 
 
-def json_chat_processor(question: str, documents: list[str]) -> list[dict[str, str]]:
```

Comment:
moving it to mark the rest as deprecated, they don't perform great anyway... could revisit in the future though so not deleting